main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md

```diff
@@ -63,7 +63,6 @@ Retrieves the omniflixhub chain object using the orchestrator.
 Creates an account on the omniflixhub chain.
 
 ## Placeholder for Delegation and Undelegation
-TODO
```

Comment:
I think we should delete these headings completely that are not explained yet, and add them when we know what to add for them.